### ISSUE TITLE
Ability to remove children and backlinks

### ISSUE THREAD

**Issue by taeboki** [dendronhq/dendron#2000]
## Please select if your request is either something new or an enhancement ##

- [x] Enhancement of an existing Feature.
- [ ] Request of a new feature.

## Please select the area your request applies to. (Multiple selections are Possible. You can leave blank if you're not sure.) ##

- [ ] Workspace - VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup - Dendron's Lookup Command
- [x] Views - Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema - Dendron Schemas
- [ ] Pod - Data import from / export to Dendron
- [x] Publish - External Site Publish
- [ ] Markdown - Markdown features, such as syntax support and features

## Is your feature request related to a problem? Please describe ##

I can see how it can really streamline how someone would be able to build their knowledgebase, but as someone who likes notes structured in a specific way, I see a way to customize our view to be a great asset to this tool. I also see some people asking whether if they can change the title/content of the children, and making it like their table of contents. This would allow users to create a much more structured note to better fit their workflow.

## Describe the solution you'd like ##

Mainly just give an option to either render/not render the child or backlinks

## Describe alternatives you've considered ##

None at the moment, seems to be no way to hide it.

**Comment by Harshita-mindfire** [comment]
Thanks @taeboki for submitting this enhancement request. I agree it would be really cool to configure it.

**Comment by aleksey-rowan** [comment]
I think these are somewhat related:

- https://github.com/dendronhq/dendron/issues/581
- https://github.com/dendronhq/dendron/issues/254

**Comment by Harshita-mindfire** [comment]
Yes, This should be configurable with `enableHierarchyDisplay` config under publishing namespace. By default, this is set to true. Setting this to false will remove the `Children` links. Click [here](https://wiki.dendron.so/notes/H9PVMguPOuZ001tNYczlh/) for more info. Similarly, the backlinks can be configured via `enableBacklinks` under publishing namespace.

**Comment by SeriousBug** [comment]
Does @Harshita-mindfire's answer resolve this issue?
